### PR TITLE
Add citation to docs

### PR DESCRIPTION
**Pull request by toloudis**
Add citation info to the docs.  Also automate the insertion of year and agave version number!

Expected time to review: 5 min

### CODE REVIEW

File: docs/agave.rst [hunk 1]
```diff
@@ -630,6 +630,29 @@ AGAVE supports the following command line options:
...
+
+Daniel Toloudis, AGAVE Contributors (|year|). AGAVE: Advanced GPU Accelerated Volume Explorer (Version |release|) [Computer software]. Allen Institute for Cell Science. https://github.com/allen-cell-animated/agave
+
+bibtex:
+
+.. parsed-literal::
+
+   @software{agave,
+    author       = {Toloudis, Daniel and AGAVE Contributors},
+    title        = {AGAVE: Advanced GPU Accelerated Volume Explorer}
```
Comment (rugeli) [inline docs/agave.rst:648]: Nit: do we want to add a comma here?

File: docs/agave.rst [hunk 2]
```diff
@@ -630,6 +630,29 @@ AGAVE supports the following command line options:
...
   Only valid in server mode on Linux. Allows AGAVE to run as a server on a headless cluster node.  On other platforms AGAVE must be run in a windowed desktop environment, even in server mode.
 
+.. _Citation:
+
+Citation
+--------
+
+.. |year| date:: %Y
+
+Daniel Toloudis, AGAVE Contributors (|year|). AGAVE: Advanced GPU Accelerated Volume Explorer (Version |release|) [Computer software]. Allen Institute for Cell Science. https://github.com/allen-cell-animated/agave
```
Comment (rugeli) [inline docs/agave.rst:640]: Do we manually update the `release` placeholder with each new version release?

File: docs/agave.rst (hunk 1 above)
Comment (toloudis) [reply docs/agave.rst]: fixed! thank you!!

File: docs/agave.rst (hunk 2 above)
Comment (toloudis) [reply docs/agave.rst:640]: Yes the release tag here picks up a version that is auto-incremented when I do releases.

**Review by rugeli — APPROVED** [review]
LGTM!

File: agave_app/citationDialog.cpp [hunk 3]
```diff
@@ -40,14 +43,16 @@ CitationDialog::CitationDialog()
...
                          "&nbsp;&nbsp;version = {" +
                          qApp->applicationVersion() +
                          "},<br/>"
                          "&nbsp;&nbsp;url       = {" +
                          agaveUrl +
-                         "}<br/>"
-                         "&nbsp;&nbsp;organization = {Allen Institute for Cell Science}<br/>"
+                         "},<br/>"
+                         "&nbsp;&nbsp;organization = {Allen Institute for Cell Science},<br/>"
                          "&nbsp;&nbsp;note      = {Computer Software}<br/>"
```
Comment (meganrm) [inline agave_app/citationDialog.cpp:56]: do you need a comma after Software} here?

**Review by meganrm — APPROVED** [review]

File: agave_app/citationDialog.cpp (hunk 3 above)
Comment (toloudis) [reply agave_app/citationDialog.cpp:56]: nope! `note` is the last one in the list

**Pull request merged**